pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/realtime_to_offline_segments/RealtimeToOfflineSegmentsTaskExecutor.java

```diff
@@ -146,20 +145,24 @@ public void preProcess(PinotTaskConfig pinotTaskConfig) {
 
     String timeColumnTransformFunction =
         configs.get(MinionConstants.RealtimeToOfflineSegmentsTask.TIME_COLUMN_TRANSFORM_FUNCTION_KEY);
-    String collectorTypeStr = configs.get(MinionConstants.RealtimeToOfflineSegmentsTask.COLLECTOR_TYPE_KEY);
-    Map<String, String> aggregatorConfigs = new HashMap<>();
+    // TODO: Rename the config key
```

Comment:
maybe introduce the new key, update the docs, and deprecate the old one?

Reply:
Yeah, planning to do it in a separate PR as this PR is focusing on the framework enhancement